Added option to select GCP and reformatted Create a Kafka instance steps

AedinC opened the pull request:
Note to reviewers:

- I reformatted and reduced the number of **Create a Kafka instance** steps to be more in line with what was originally saved in the commented out part of that section. I think this works better especially once new values in the **Create a Kafka instance** form become available (billing for example). 
- HTML preview available [here](http://file.ams.redhat.com/aecollin/JK-9297/README.html).
- I need to get a new screenshot of the updated UI. I am in contact with UXD about this so this PR will be in draft until I get that.

jbyrne-redhat commented on docs/kafka/getting-started-kafka/README.adoc:
```diff
@@ -121,11 +121,15 @@ ifndef::qs[]
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click *Create Kafka instance*.
-. Enter a unique name for the Kafka instance, such as `my-first-kafka-instance`.
-. From the *Cloud region* drop-down menu, select the relevant option. The selection for availability zones is preconfigured as *Single* for a trial instance and *Multi* for a production instance.
+. On the *Kafka Instances* page of the {product-kafka} web console, click *Create Kafka instance*.
```
In guides such as Managing account access, we hyperlinked 'Kafka Instances':

```
On the {service-url-kafka}[_Kafka Instances_^] page of the {product-kafka} web console...
```

jbyrne-redhat commented on docs/kafka/getting-started-kafka/README.adoc:
```diff
@@ -121,11 +121,15 @@ ifndef::qs[]
-. Enter a unique name for the Kafka instance, such as `my-first-kafka-instance`.
-. From the *Cloud region* drop-down menu, select the relevant option. The selection for availability zones is preconfigured as *Single* for a trial instance and *Multi* for a production instance.
+. On the *Kafka Instances* page of the {product-kafka} web console, click *Create Kafka instance*.
+. Complete the *Create a Kafka instance* form to define the following instance details. Some values currently have only one option.
```
I'm not sold on the phrase "Some values currently have only one option".
-  Using _currently_ implies to users that this will change in the future. While that might be true, we don't want to make those kinds of promises.
- I think I'd avoid using the term _value_ to represent an instance detail. To me, a _value_ is the number you type in a field or choose from a list. Maybe "Some instance details have only one available option"?

jbyrne-redhat replied:
Hey, @AedinC. Can you consider applying the comments I previously made here?

AedinC replied:
Yes, I seem to have rewritten over the second commit that incorporated your feedback! Will make those changes again.

jbyrne-redhat commented on docs/kafka/getting-started-kafka/README.adoc:
```diff
@@ -121,11 +121,15 @@ ifndef::qs[]
+. Complete the *Create a Kafka instance* form to define the following instance details. Some values currently have only one option.
+* *Instance name*: Enter a unique name for the Kafka instance, such as `my-first-kafka-instance`.
+* *Cloud provider*: Accept the default value of *Amazon Web Services* or select *Google Cloud Platform*.
+* *Cloud region*: Select the relevant region from the list.
```
I never loved our use of _relevant_ here. I suggest "Select a cloud provider region from the list".

jbyrne-redhat replied:
Love what you did here to streamline the steps for creating an instance! :+1:

jbyrne-redhat replied:
Ditto with my previous comments here.

jbyrne-redhat reviewed (approved):
I reviewed what you have so far, Aedin. These changes look good - I particularly like what you did to streamline the instance creation steps. Just a few small comments for you to consider.

AedinC commented:
> I reviewed what you have so far, Aedin. These changes look good - I particularly like what you did to streamline the instance creation steps. Just a few small comments for you to consider.

Thanks John for the great feedback as always.

AedinC commented:
> I reviewed what you have so far, Aedin. These changes look good - I particularly like what you did to streamline the instance creation steps. Just a few small comments for you to consider.

Hi @jbyrne-redhat. Would you mind looking at this again whenever you get a chance? GCP is only going to be available for trial instances this side of Christmas so I have modified the text slightly and have left the create Kafka instance screenshot as is but modified the title to include 'production'.  Duncan has given a LGTM in the corresponding JIRA ticket.

jbyrne-redhat commented:
> > I reviewed what you have so far, Aedin. These changes look good - I particularly like what you did to streamline the instance creation steps. Just a few small comments for you to consider.
> 
> Hi @jbyrne-redhat. Would you mind looking at this again whenever you get a chance? GCP is only going to be available for trial instances this side of Christmas so I have modified the text slightly and have left the create Kafka instance screenshot as is but modified the title to include 'production'. Duncan has given a LGTM in the corresponding JIRA ticket.

Hey, @AedinC. Would you mind updating your HTML preview?

jbyrne-redhat commented:
> > > I reviewed what you have so far, Aedin. These changes look good - I particularly like what you did to streamline the instance creation steps. Just a few small comments for you to consider.
> > 
> > 
> > Hi @jbyrne-redhat. Would you mind looking at this again whenever you get a chance? GCP is only going to be available for trial instances this side of Christmas so I have modified the text slightly and have left the create Kafka instance screenshot as is but modified the title to include 'production'. Duncan has given a LGTM in the corresponding JIRA ticket.
> 
> Hey, @AedinC. Would you mind updating your HTML preview?

NVM - I see that it is! :smile:

AedinC commented:
> > > I reviewed what you have so far, Aedin. These changes look good - I particularly like what you did to streamline the instance creation steps. Just a few small comments for you to consider.
> > 
> > 
> > Hi @jbyrne-redhat. Would you mind looking at this again whenever you get a chance? GCP is only going to be available for trial instances this side of Christmas so I have modified the text slightly and have left the create Kafka instance screenshot as is but modified the title to include 'production'. Duncan has given a LGTM in the corresponding JIRA ticket.
> 
> Hey, @AedinC. Would you mind updating your HTML preview?

Hi @jbyrne-redhat - I did update it.

> > > I reviewed what you have so far, Aedin. These changes look good - I particularly like what you did to streamline the instance creation steps. Just a few small comments for you to consider.
> > 
> > 
> > Hi @jbyrne-redhat. Would you mind looking at this again whenever you get a chance? GCP is only going to be available for trial instances this side of Christmas so I have modified the text slightly and have left the create Kafka instance screenshot as is but modified the title to include 'production'. Duncan has given a LGTM in the corresponding JIRA ticket.
> 
> Hey, @AedinC. Would you mind updating your HTML preview?

Hi @jbyrne-redhat I did update it.

jbyrne-redhat commented on docs/kafka/getting-started-kafka/README.adoc:
```diff
@@ -154,10 +145,15 @@ NOTE: The cloud provider is preconfigured to the same value for both trial and p
 . Click *Create instance* to start the creation process for your Kafka instance.
 +
 --
+[.screencapture]
+.Kafka production instance configuration details
+image::sak-configure-kafka-instance.png[Image of instance configuration details in Create Kafka instance window]
+
```
Given that we've omitted this image from step 1, I don't think we actually need it anymore. I wouldn't put it in this step.

AedinC replied:
@jbyrne-redhat Did we agree to leave this image out from the customer portal version of the guide? If so, I won't need to get an updated screenshot.

jbyrne-redhat replied:
@AedinC I haven't been in any conversations where we "agreed" to do this, per se. But my opinion is that we can do without it. It creates ongoing maintenance, and I don't think it adds a ton of value. 

An alternative would be to restore it to step 1 (as it is [currently](https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/f351c4bd-9840-42ef-bcf2-b0c9be4ee30a#_b4f95791-b992-429d-9e8e-cceb63ae829f)) and reword step 2 here  as "Define the following instance details" or "Define the instance details shown in the following list." (don't mention the name of the form, in other words).

I'm inclined towards option 1, so that we don't have to keep updating this figure every time the UI has a slight change.

AedinC replied:
@jbyrne-redhat I do think we could add more images to our guides but this one adds no value that I can see so I'm going to remove it.

jbyrne-redhat reviewed (changes requested):
Thanks for making these updates, @AedinC. Just a couple of suggested updates.

jbyrne-redhat commented on docs/kafka/getting-started-kafka/README.adoc:
```diff
@@ -127,24 +127,15 @@ ifndef::qs[]
+. On the *Kafka Instances* page of the {product-kafka} web console, click *Create Kafka instance*.
+. Complete the *Create a Kafka instance* form to define the following instance details. Some values currently have only one option.
+* *Instance name*: Enter a unique name for the Kafka instance, such as `my-first-kafka-instance`.
+* *Cloud provider*: Accept the default value of *Amazon Web Services*. If you are creating a trial instance, you can select *Google Cloud Platform*.
```
Suggest: "For a production Kafka instance, accept the default value of *Amazon Web Services*. If you're creating a trial instance, you also have the option to select *Google Cloud Platform*."

jbyrne-redhat commented on docs/kafka/getting-started-kafka/README.adoc:
```diff
@@ -127,24 +127,15 @@ ifndef::qs[]
 
 .Procedure
-. In the {product-kafka} {service-url-kafka}[web console^], select *Kafka Instances* and click *Create Kafka instance*.
+. On the *Kafka Instances* page of the {product-kafka} web console, click *Create Kafka instance*.
```
Per our style guidelines, hyperlink "web console".

jbyrne-redhat commented on docs/kafka/getting-started-kafka/README.adoc:
```diff
@@ -127,24 +127,15 @@ ifndef::qs[]
+* *Instance name*: Enter a unique name for the Kafka instance, such as `my-first-kafka-instance`.
+* *Cloud provider*: For a production Kafka instance, accept the default value of *Amazon Web Services*. If you're creating a trial instance, you also have the option to select *Google Cloud Platform*.
+* *Cloud region*: Select a cloud provider region from the list.
+* *Availability zones*: The selection for availability zones is preconfigured as *Single* for a trial instance and *Multi* for a production instance.
```
Think we should flip this description to state the production instance value first, like we do for Cloud provider.

"The selection for availability zones is preconfigured as *Multi* for a production instance and *Single* for a trial instance."

jbyrne-redhat reviewed (approved):
Two final, small comments, @AedinC. Otherwise approved.

AedinC commented:
> Two final, small comments, @AedinC. Otherwise approved.

Thanks @jbyrne-redhat.
